README.md

```diff
@@ -21,7 +21,7 @@ Beadledom is made up of the following components:
 * [lifecycle-governator](lifecycle-governator#beadledom-lifecycle-governator) - An implementation of the [beadledom-lifecycle](lifecycle#beadledom-lifecycle) using [Governator](https://github.com/Netflix/governator).
```

Comment:
>Should the swagger2 module, as in our second implementation of swagger-core, be renamed to just swagger since it could be confused with using swagger-core 2 when we are on swagger-core 1.5?

Yes, it is very confusing... I would add a comment to the README of the project that the project supports 2.0 of OpenAPI Spec. We should also add that information to the docs module swagger2.rst file.

Reply:
124eaa44

Reply:
c58b5af4